packages/perseus-editor/src/widgets/__docs__/categorizer-editor.stories.tsx

```diff
@@ -2,33 +2,33 @@ import {ApiOptions} from "@khanacademy/perseus";
 import * as React from "react";
 import {action} from "storybook/actions";
 
+import {question1} from "../../../../perseus/src/widgets/categorizer/categorizer.testdata";
```

Comment:
I think you're supposed to avoid accessing other packages directly like this. I've seen this before though, so I'm asking in Slack about it right now.